Makefile

```diff
@@ -1,25 +1,28 @@
+OPAM_PREFIX?=$(DESTDIR)$(shell opam config var prefix)
+OPAM_LIBDIR?=$(DESTDIR)$(shell opam config var lib)
```

Comment:
Does this require a specfile change?

Reply:
No, this comes from a patch in `xenserver-specs` that we can now remove (the removal is part of the merge PR there)